server/src/service-layer/controllers/UserSignup.ts

```diff
@@ -0,0 +1,45 @@
+import AuthService from "business-layer/services/AuthService"
+import UserDataService from "data-layer/services/UserDataService"
+import { UserSignupBody } from "service-layer/request-models/UserSignupRequests"
+import { UserSignupResponse } from "service-layer/response-models/UserSignupResponse"
+import { Body, Controller, Post, Route, SuccessResponse } from "tsoa"
+
+@Route("signup")
+export class UserSignup extends Controller {
+  @Post()
+  @SuccessResponse(200, "Signup successful")
+  // return a JWT token at the end
+  public async signup(
+    @Body() requestBody: UserSignupBody
+  ): Promise<UserSignupResponse> {
+    const userService = new UserDataService()
+    const authService = new AuthService()
+
+    // Received userInfo omits membership and stripe_id
+    const userInfo = requestBody.user
+
+    try {
+      // Create user data in Auth Service
+      const userRecord = await authService.createUser(requestBody.email)
```

Comment:
nit: i would still have a separate try catch so we can return `{error: email in use}` etc

Reply:
Ah yes, I remember thinking about that conflict before but somehow deleted it some while ago. Ty! Will implement.

Reply:
actually ignore that, in the 500 response you can just make error = the caught error

Reply:
Oops, no worries, just created another try/catch for the different 409 conflict.